README.md

```diff
@@ -11,6 +11,25 @@ Start by cloning the repository with `git clone --recursive https://github.com/T
 
 If the repository was cloned non-recursively previously, use `git submodule update --init` to clone the necessary submodules.
 
+## Building the project
+1. Run the [Setup.bat](https://github.com/TheCherno/Hazel/blob/master/scripts/Setup.bat) file found in `scripts` folder. This will download the 'VulkanSDK.exe' installation file.
+2. If VulkanSDK is not installed, the script will execute the VulkanSDK.exe file, and will prompt the user to install the SDK.
```

Comment:
I would not specify what the script downloads, as this may change over time (think about downloading premake using the setup script).
```suggestion
1. Run the [Setup.bat](https://github.com/TheCherno/Hazel/blob/master/scripts/Setup.bat) file found in `scripts` folder. This will download the required prerequisites for the project if they are not present yet.
2. One prerequisite is the VulkanSDK. If it is not installed, the script will execute the VulkanSDK.exe file, and will prompt the user to install the SDK.
```

Reply:
Alright, will do that.